packages/framework/src/Model/Pricing/BasePriceCalculation.php

```diff
@@ -72,6 +70,8 @@ public function calculateBasePriceRoundedByCurrency(
     }
 
     /**
+     * @deprecated method is deprecated and will be removed in the next major. This method is not used, only in tests
```

Comment:
It would be great to added a method `@trigger_error` into method's body. Or not?

Reply:
Why not, fixed.

Reply:
Thanks to fix. Why do I think it's good to have it there? Because phpunit bridge logs occurrence of trigger_error function :smile: